From a51f1ad4a5e7176ed9940cb76edd9da88349a81c Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@st.com>
Date: Fri, 22 Apr 2022 17:22:29 +0200
Subject: [PATCH 0410/1695] plat-stm32mp2: conf: support HSEM driver

Default enable HSEM driver for platform stm32mp2.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 4c936cab0a64f1335b38728c945a9b45227cd29f plat-stm32mp2: conf: support HSEM driver]
Signed-off-by: Gatien Chevallier <gatien.chevallier@st.com>
Change-Id: I8ebba04ef7e4441672304e3c3246787480dc5681
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/249375
Reviewed-by: Lionel DEBIEVE <lionel.debieve@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 710a26de6..0b09787f5 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -71,6 +71,7 @@ CFG_STM32_EXTI ?= y
 CFG_STM32_FMC ?= y
 CFG_STM32_GPIO ?= y
 CFG_STM32_HPDMA ?= y
+CFG_STM32_HSEM ?= y
 CFG_STM32_IAC ?= y
 CFG_STM32_OMM ?= y
 CFG_STM32_RIF ?= y
-- 
2.47.2

